src/app/notification-manager/components/NotificationManager.spec.ts

```diff
@@ -44,9 +21,7 @@ describe('NotificationManager.vue', () => {
       },
     })
 
-    const openModalButton = wrapper.find('[data-testid="open-modal-button"]')
-    expect(openModalButton.html()).toContain('Check your mesh!')
-    await openModalButton.trigger('click')
+    await store.dispatch('notifications/openModal')
```

Comment:
Would be nice to have an e2e test to ensure clicking the little bell shows this modal. Maybe a follow up.